README.md

```diff
@@ -1 +1,42 @@
-# 500apps-public-gtm-ninjachat
\ No newline at end of file
+# Ninjachat
+[Get Started](https://infinity.500apps.com/ninjachat) with with [Ninjachat](https://ninjachat.com)
+
+A live chat application with complex features and also with a high ease of use quotient. Ninjachat has all the features one needs to increase and improve customer engagement. Personalise the chat widgets, connect with visitors and respond to them in real time. Ninjachat helps boost customer retention with no glitches thereby providing a good user experience.
+Create departments and install the script generated in your websites to run the Ninjachat widget. Generate leads, resolve customer complaints, address issues, etc with easy-to-use Ninjachat application. As the customer traction of your website grows, you can add more agents to the application, where each agent is equipped with handling 25 simultaneous live chats.You can also restrict the number of live chats an agent can do, there by giving time for new agents to get trained before scaling up the operations. Customise user experience tracking live chats and by building custom short cuts as well.
+
+- [Get Started](https://infinity.500apps.com/ninjachat) with easy to use and configurable live chat application which can be configured not only to websites but also to social media handles like LinkedIn , Facebook , etc .[Ninjachat](https://ninjachat.com)
+
+
+- Sign in to your Google account.
+
+- Go to the Google Tag Manager home and click on the Create Account button on the right
+
+- Enter the account as (Account name) and container name( Website URL) . The container holds all your tags, such as the Google Analytics tag, Google Ads tag, or ChatBot tag. Choose the target platform to web and click on the Continue button.
+
+- Read through the GTM Terms of Service. Click Yes to accept them and continue.
+
+You will get a container with a code snippet. Copy the code and paste it into every page of your site in the recommended places.
+Click Ok to close the code snippet box and finish setting up the GTM manager and your first container.
+
+Adding the ChatBot tag to your website
+Once you're logged in to your account with the container, it's time to add a new tag with the ChatBot tracking code inside it. To do so, follow these steps   
+In the Google Tag Manager Home, click on the New Tag tile.
+Click on the Tag Configuration tile and select Custom HTML from the tags list.
+Paste the ChatBot tracking code to the HTML field.
```

Comment:
should it be just api key. Why the complete scripting code?

Reply:
@subramaniyamkolluri Sir.Above content is given by @indira. Previoulsy she gaves content for botup also
https://github.com/agilecrm/500apps-public-gtm-botup/pull/1/files

Reply:
Modified Sir